changelog

```diff
@@ -0,0 +1,34 @@
+* Fri Oct 02 2015 Francisco Romero <francisco.romerobueno@telefonica.com> 0.1.0
```

Comment:
Yes. Since I don't have a let's say "regular" changelog within the neore folder, I will accumulate the changes in this file in the meantime.